boostnote.json

```diff
@@ -1,9 +1,9 @@
 {
     "homepage": "https://boostnote.io/",
     "license": "GPL-3.0-or-later",
-    "version": "0.11.7",
-    "url": "https://github.com/BoostIO/boost-releases/releases/download/v0.11.7/BoostnoteSetup.exe#/dl.7z",
-    "hash": "d4df317aabca17cdc62c56deaf934ddc223735f777d5d3db6d1b68f55fd5b6f0",
+    "version": "0.11.8",
+    "url": "https://github.com/BoostIO/boost-releases/releases/download/v0.11.8/BoostnoteSetup.exe",
```

Comment:
I think we need to leave the `#/dl.7z` on the URL

Reply:
Ahh, yes. I missed that the autoupdate didn't include the `#/dl.7z` part. I'll push a fix in a few seconds.